CODE/style.css

```
@@ -0,0 +1,130 @@
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #d2d3d3;
+
+}
+
+/* pull QUote*/
+blockquote {
+    text-align: left;
+    margin: 0;
+}
+
+
+header {
+    background-image: url(images/maxresdefault.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+/*Title and Author*/
+h1 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 60px;
+    text-align: center;
+    color: #ce5454f8;
```

Hex values should start with a pound sign (`#`) and be six characters long; do you perhaps mean for this to be
```suggestion
    color: #c95252;
```

Note that you use this value in other places and will need to correct or update it in all instances.